packages/ui/libs/ui/core/src/containers/GenericForm/GenericForm.types.ts

```diff
@@ -14,10 +14,10 @@ import { DeepPartial, UnpackNestedValue, UseFormReset } from 'react-hook-form';
 import { DatePickerProps } from '@material-ui/lab';
 
 export type GenericFormField = {
-  disabled?: boolean;
+  frozen?: boolean;
   name: string;
-  label: string;
-  type?: string;
+  label: string | null;
+  type?: 'text' | 'password';
```

Comment:
This shouldn't be here, because it will be imposible to use other types, thus making this component non-generic